src/sass/layout/_site-footer.scss

```
@@ -1 +1,47 @@
-.site-footer {}
+.site-footer {
+  align-content: flex-end;
+  background-color: black;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;   
+  padding:{ 
+    
+    bottom: 10px;
+    top: 40px;
+  }
+  
+   @media (max-width: 618px) {
+        margin: 0 auto;
+        display: block;        
+     
+  }
+  
+
+.footer-hover:hover{
+    color: yellow;
+  }
+  
+.list-contact{
+  display: block; 
+  color: white; 
+  list-style-type: none;
+  padding: 0;
+  padding-left: 70px;  
```

In general, you want to avoid writing CSS like this -- where you write one rule only to then immediately overwrite it. By default, a list _only_ has padding on the left side, so the rule on line 29 is sufficient:
```suggestion
  padding-left: 70px;  
```